nautobot/extras/tables.py

```diff
@@ -1136,7 +1143,8 @@ class Meta(BaseTable.Meta):
 
 class AssociatedContactsTable(StatusTableMixin, RoleTableMixin, BaseTable):
     pk = ToggleColumn()
-    contact_or_team = tables.TemplateColumn(CONTACT_OR_TEAM, verbose_name="Contact/Team")
+    contact_type = tables.TemplateColumn(CONTACT_OR_TEAM_ICON, verbose_name="Contact Type")
```

Comment:
Django tables wants to make this column huge. I hacked something together to make it a little better looking, what do you think?

### Before

![image](https://github.com/nautobot/nautobot/assets/75227981/3dc17de7-f3bc-4325-a202-7a9b110af791)


### After
![image](https://github.com/nautobot/nautobot/assets/75227981/7ec0b4ff-9403-4a1e-82fe-2f99fa7e5f6e)

```py
    contact_type = tables.TemplateColumn(
        CONTACT_OR_TEAM_ICON,
        verbose_name="Contact Type",
        attrs={"td": {"style": "width:20px;"}, "th": {"style": "white-space: nowrap;"}},
    )
```

Reply:
Looks good to me but let's use `verbose_name="Type"` to make it even less space-consuming.

Reply:
If we do that we can get rid of `"th": {"style": "white-space: nowrap;"}`